controls/raddiagram/features/global-settings.md

```diff
@@ -12,34 +12,49 @@ position: 15
+	
+	> This property is obsolete. Use __RouterInflationValue__ instead.
 
-* __MaximumZoom__ - The maximum zoom level. The default value of this constant is 5. Read more about zooming in the [Pan and Zoom](585daa93-6cbd-47a4-8fd3-61469984b67e#Zooming) tutorial.			
+* __RouterInflationValue__: Gets or sets the distance around the shapes which the router inflates the shapes with, if implemented. This property is used by the __InflatedRectRouted__ and __AStarRouter__.
+	
+* __ZoomScaleFactor__: Gets or sets the default zoom factor that is used as a multiplier or divider to produce the next step in a zoom operation. The default value of this constant is 1.2. Read more about zooming in the [Pan and Zoom](585daa93-6cbd-47a4-8fd3-61469984b67e#Zooming) tutorial.			
 
-* __MinimumZoom__ - The minimum zoom level. The default value of this constant is 0.5. Read more about zooming in the [Pan and Zoom](585daa93-6cbd-47a4-8fd3-61469984b67e#Zooming) tutorial.			
+* __MaximumZoom__: Gets or sets the maximum zoom level. The default value of this constant is 5. Read more about zooming in the [Pan and Zoom](585daa93-6cbd-47a4-8fd3-61469984b67e#Zooming) tutorial.			
 
-* __RotationAngleSnap__ - Gets or sets the snapping angle when rotating items. The default value of this constant is 5 degrees. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
+* __MinimumZoom__: Gets or sets the minimum zoom level. The default value of this constant is 0.5. Read more about zooming in the [Pan and Zoom](585daa93-6cbd-47a4-8fd3-61469984b67e#Zooming) tutorial.			
 
-* __RotationHitTestRadius__ - gets or sets the radius around the rotation thumb where a click operation activates the thumb. The default value of this constant is 7px. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
+* __RotationAngleSnap__: Gets or sets the snapping angle when rotating items. The default value of this constant is 5 degrees. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
 
-* __RoutingGridSize__ - gets or sets the routing grid cell size. The value of this constant is used in the layout algorithm used to layout the __RadDiagramConnections__. The default value of this constant is 40px.			
+* __RotationHitTestRadius__: Gets or sets the radius around the rotation thumb where a click operation activates the thumb. The default value of this constant is 7px. Read more about rotation in the [Rotation]({%slug raddiagrams-features-rotation%}) tutorial.			
 
-* __SelectionHitTestRadius__ - The radius of the area around each item which is considered selectable. The default value of this constant is 10px.			
+* __RoutingGridSize__: Gets or sets the routing grid cell size. The value of this constant is used in the layout algorithm used to layout the __RadDiagramConnections__. The default value of this constant is 40px.			

```

Comment:
"the layout algorithm used to layout..." probably can be replaced with "the algorithm used to layout..."